core/Pos/Util/Chrono.hs

```diff
@@ -37,7 +39,10 @@ newtype OldestFirst f a = OldestFirst {getOldestFirst :: f a}
             Binary, Bi,
             Arbitrary)
 
+makePrisms  ''NewestFirst
 makeWrapped ''NewestFirst
+
+makePrisms  ''OldestFirst
```

Comment:
It's more clear than `_Wrapped`, you get to see the name of the constructor at call site.